org.adempiere.base/src/org/compiere/model/MProcessPara.java

```diff
@@ -373,6 +378,36 @@ && getAD_Element_ID() != 0 && isCentrallyMaintained())
 			if (Util.isEmpty(p.getClassname()) && Util.isEmpty(p.getProcedureName()) && Util.isEmpty(p.getJasperReport()))
 				setIsShowNegateButton(true);
 		}
+		
+		if(getValueMin()!=null) {
+			try {
+				if(getAD_Reference_ID()==15) { // Date

```

Comment:
please use here DisplayType.Date instead of the hardcoded 15, same below for the other references.